### ISSUE TITLE
Wait for MP and Mongo

### ISSUE THREAD

**Issue by blootsvoets** [RADAR-CNS/RADAR-RestApi#66]
In the Dockerfile, at initialisation, wait until MP and MongoDB are ready before starting Tomcat. Alternatively, do the wait inside the RADAR servlet.

**Comment by blootsvoets** [comment]
Using Grizzly server instead now - a wait is no longer needed.
